apps/meteor/app/apps/server/bridges/livechat.ts

```diff
@@ -89,7 +90,7 @@ export class AppLivechatBridge extends LivechatBridge {
 
 		let agentRoom: SelectedAgent | undefined;
 		if (agent?.id) {
-			const user = await Users.getAgentInfo(agent.id);
+			const user = await Users.getAgentInfo(agent.id, settings.get('Livechat_show_agent_email'));
```

Comment:
Is it possible to use the `Settings` raw model here? We're working on removing these fiber usages on our Microservices PRs

Reply:
Hey! There shouldn't be a problem, since `settings.get` doesn't rely on fibers since some time ago (it actually never _relied_ on them, but there was a kind of middleware that used fibers, which was removed)

Reply:
If you find any problems after this, call Diego :trollface:

Reply:
Oh, ok, cool!